src/Npgsql/ConnectorPool.cs

```diff
@@ -129,7 +129,7 @@ internal ConnectorPool(NpgsqlConnectionStringBuilder settings, string connString
                 throw new ArgumentException($"Connection can't have ConnectionIdleLifetime {connectionIdleLifetime} under ConnectionPruningInterval {_pruningSamplingInterval}");
 
             _pruningTimer = new Timer(PruningTimerCallback, this, Timeout.Infinite, Timeout.Infinite);
-            _pruningSampleSize = DivideRoundingUp(connectionIdleLifetime.Seconds, pruningSamplingInterval.Seconds);
+            _pruningSampleSize = DivideRoundingUp(connectionIdleLifetime.TotalSeconds, pruningSamplingInterval.TotalSeconds);
```

Comment:
`DivideRoundingUp` accepts `int`s as parameters.

Reply:
Right... I'll just use the original inputs as they're already seconds with integer precision. I'm not sure why Rider isn't picking up on this. Only after retyping the first I got an error, but only on the first argument.

<img width="949" alt="Screenshot 2020-01-13 at 17 06 47" src="https://user-images.githubusercontent.com/4218809/72271455-36d87000-3627-11ea-8d93-9aedff4d2628.png">